js/scripts.js

```diff
@@ -0,0 +1,10 @@
+﻿var text = 'Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.';
+var animal = 'Zielone słonie';
+var animalUpperCased = animal.toUpperCase();
+var textCharsAfter = text.replace('Papugi', animalUpperCased);
+var newCharsAfter = textCharsAfter.substr(0, 45);
```

Comment:
Twój program zadziała tylko dla tego konkretnego zdania. Jeśli `text` ulegnie zmienie, to już nie wyświetlisz idealnie połowy tekstu. Spróbuj zrobić to tak żeby tekst dzielić na pół bez względu na to jak długi on będzie.